pai-experimental/cdp.sol

```diff
@@ -348,5 +371,30 @@ contract CDP is DSMath, DSNote, Template {
         CDPRecord storage data = CDPRecords[record];
         return (data.collateral, data.accumulatedDebt1, rmul(data.accumulatedDebt1, accumulatedRates1), data.accumulatedDebt2, rmul(data.accumulatedDebt2, accumulatedRates2));
     }
-    
+
+    function readCDP(uint record) public view returns(address, uint, uint, uint){
+        return (CDPRecords[record].owner,CDPRecords[record].collateral,CDPRecords[record].accumulatedDebt1,CDPRecords[record].accumulatedDebt2);
+    }
+
+    function getFee() public view returns(uint){
+        return stabilityFee;
+    }
+
+    function getTax() public view returns(uint){
+        return governanceFee;
+    }
```

Comment:
to follow SAI conventions, fee -> governance fee and tax -> stability fee

Reply:
don't mind any method after '///only for debug'. I assume all these methods will be deleted after detailed testing.